.github/workflows/test-mongodb.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: test with mongodb
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        entry:
+          - { ruby: 2.6.2, mongoid: 6.4.8, mongodb: 4.4 }
+          - { ruby: 2.6.2, mongoid: 7.2.3, mongodb: 4.4 }
+          - { ruby: 2.6.2, mongoid: 7.3.0, mongodb: 4.4 }
+    name: test (ruby=${{ matrix.entry.ruby }}, mongoid=${{ matrix.entry.mongoid }}, mongodb=${{ matrix.entry.mongodb }})
```

Comment:
See if we can add 3.1.0 and MongoDB 5? Also fine in another PR.

Reply:
Could you redefine the versions matrix here again? I'm afraid it might become cumbersome if I add mongo v5 for all the ruby & mongoid versions.  
By redefine, I mean for example,
```
ruby 2.6.2 - mongoid 7.3.0 - mongodb 4.4
ruby 2.6.2 - mongoid 7.3.0 - mongodb 5.0
ruby 2.7.2 - mongoid 7.3.0 - mongodb 5.0
```
Something like that.  
Once it's confirmed, I will run the tests on my local and draft the PR ASAP.  
Thanks!

Reply:
@dblock ^

Reply:
I don't think we need all the combinations. I would just ensure some reasonable coverage. Feel free to make changes as you see fit.

Reply:
Got you, thanks!